manual/TypeInspection.rst

```diff
@@ -154,20 +177,23 @@ configuration file with this content:
     }
   }
 
-This clearly is more verbose than it needs to be. By also providing an overload
-for ``inspect_value``, we can teach CAF how to inspect an ID directly as a
-*value* without having to recursively visit it as an object:
+This clearly appears more verbose than it needs to be. Users generally need not
+care about such internal types like ``id`` that only exist as a safeguard during
+programming.
+
+Hence, we generally recommend making such types transparent to CAF inspectors.
+For our ``id`` type, the ``inspect`` overload may instead look as follows:
 
 .. code-block:: C++
 
   template <class Inspector>
-  bool inspect_value(Inspector& f, id& x) {
-    return f.apply_value(x.value);
+  bool inspect(Inspector& f, id& x) {
+    return f.apply(x.value);
   }
 
-With this overload in place, inspectors can now remove one level of indirection
-and read or write strings whenever they encounter an ``id`` as value. This
-allows us to simply our config file from before:
+With this implementation instead of the previous one, inspectors simply read or
+write strings whenever they encounter an ``id`` as value. This simplifies our
```

Comment:
I'm not entirely sure what you want to say, but the "as value" confuses me. Maybe this?

```suggestion
write the strings as values whenever they encounter an ``id``. This simplifies our
```